src/main/java/analyzer/exercises/loglevels/LogLevelsAnalyzer.java

```diff
@@ -43,6 +44,10 @@ public void analyze(Solution solution, OutputCollector output) {
 
     @Override
     public void visit(MethodDeclaration node, OutputCollector output) {
+        if (!METHODS_TO_ANALYZE.contains(node.getNameAsString())) {
+            return;
+        }
```

Comment:
Note that this bypasses _all_ checks on helper methods. I'm not sure that is a good idea to do by default. For example, consider a solution that contains hard-coded test case data in a helper method instead of one of the main methods, I think you want to catch those right?

Reply:
I kinda think is an overkill to analyzer helper method as well, mainly because they can be a sort of test to validate the other methods of their code or something like that, in the example of the user that opened this issue you can see that the actual `result` method wasn't use in any other part of the code but it was triggering a comment